server.js

```diff
@@ -4739,6 +4739,53 @@ cache(function(data, match, sendBadge, request) {
   }
 }));
 
+// Issue Stats integration.
+camp.route(/^\/issuestats\/([^\/]+)\/(.+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var host = match[1];      // e.g. `github`
+  var userRepo = match[2];  // e.g. `ruby/rails`
+  var type = match[3];      // e.g. `issue` or `pr`
+  var format = match[4];
+
+  var badgeData = getBadgeData('Issue Stats', data);
+
+  var url = 'http://issuestats.com/' + host + '/' + userRepo;
+  var qs = {format: 'json'};
+  if (data.hasOwnProperty('concise')) {
```

Comment:
I was trying to stay consistent with the issuestats URL format, which uses this query string parameter, but I'll change it to a path component as you suggested.  Thanks.